ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/CentOS.yml

```diff
@@ -50,6 +50,16 @@
     - ansible_distribution_major_version == "6"
   tags: build_tools
 
+- name: Install repos for devtoolset-7 on CentOS6 x86_64
+  yum:
+    name: centos-release-scl
+  state: installed
+  when:
+    - ansible_distribution == "CentOS"
+    - ansible_architecture == "x86_64"
+    - ansible_distribution_major_version == "6"
```

Comment:
This repo is specific to Cent6?  And x86?  Because the packages listed below will install for all Cent releases and architectures.

Reply:
As per [Developer Toolset 7](https://www.softwarecollections.org/en/scls/rhscl/devtoolset-7/) is specific to CentOS 6, 7 x86-64.
This should address CentOS 6 x86-64 only .